src/components/Autosuggest/Autosuggest.stories.tsx

```diff
@@ -104,6 +104,18 @@ const LibraryExample = ({ renderInputComponent }) => {
     onChange,
   };
 
+  const renderInputComponentWithLabel = inputProps => {
+    return (
+      <div>
```

Comment:
Consider using fragment here to avoid an unnecessary container.

Reply:
@nypl-wluisi Can we close out this PR?

Reply:
@bigfishdesign13 Yeah we can close. It might be worth creating a Jira ticket to investigate if the DS should be adding `display: flex` to Button component, as it creates some issues for consuming apps the component.